config/locales/cy.yml

```diff
@@ -69,6 +69,10 @@ cy:
       employer_validation:
         blank: Rhowch gyfraniad eich cyflogwr
         out_of_range: Rhaid i gyfraniad eich cyflogwr fod yn yr ystod 0 - 100%
+      manually_optIn: |
```

Comment:
I agree, I chose `manually optIn` to be consistent with `callout__optIn` on line 52, but I will change